src/main/deltachat/chatlist.js

```diff
@@ -96,6 +96,10 @@ function getSmallChatById (chatId, list, i) {
   const summary = list.getSummary(i).toJson()
   const lastUpdated = summary.timestamp ? summary.timestamp * 1000 : null
 
+  if (summary.text2 === '[The message was sent with non-verified encryption.. See "Info" for details.]') {
```

Comment:
Isn't there a better way to do this? some msg error enum or the like?

Reply:
The core doesn't give us anything better

Reply:
did we already made an feature request there?

Reply:
Nope but no other client currently translates this string into another string. This is a error and normally shouldn't happen.

Reply:
@nicodh i think translating this error message doesn't really make sense as it shouldn't happen in the first case. @hpk42 @r10s any opinions on this?

Reply:
Yes sure - just a temporary workaround

Reply:
And we have another not translated error message: [foo@bar is not a member of this verified group. See "Info" for details.]
So maybe better wait still the core translates these messages? Or gives a flag or something so we can adapt the UI

Reply:
I don't think we need to translate those errors as they shouldn't happen. It's more to trace bugs down as far as i understand this.